README.md

```diff
@@ -54,74 +54,50 @@ The generated docs can then be found locally at
 
 ## Using `odoc` with BuckleScript/Reason
 
-> BuckleScript support currently requires the latest development version of odoc.
-
-While the BuckleScript/Reason toolchain relies on `npm`, `odoc` at the moment
-needs to be used from a working OCaml toolchain.
-
-This means we follow the same installation than above, but using the
-`4.02.3+buckle-master` version of the OCaml compiler.
+You can use the [`bsdoc`](https://ostera.github.io/bsdoc) npm package to use
+`odoc` in your BuckleScript projects.
 
 ```sh
-λ opam switch 4.02.3+buckle-master
-λ eval `opam config env`
-λ opam pin add odoc.dev git+https://github.com/ocaml/odoc
-
+$ cd MyProject
```

Comment:
Given that the instructions are already provided in [[bsdoc]'s README](https://github.com/ostera/bsdoc#getting-started), I don't think it's necessary to repeat them here. Each tool should be responsible for its own documentation, otherwise things will get out of date eventually.